app/src/controllers/itemsHistory/actionCreators.js

```diff
@@ -0,0 +1,35 @@
+import {
+  FETCH_ITEMS_HISTORY,
+  SET_ITEMS_HISTORY,
+  FETCH_HISTORY_PAGE_INFO,
+  SET_VISIBLE_ITEMS_COUNT,
+  RESET_HISTORY,
+  RESET_FETCH_HISTORY,
+} from './constants';
+
+export const fetchItemsHistoryAction = (payload) => ({
```

Comment:
It's unclear what whould be passed as an argument. Please rename or use destructirung of it's an object